var/spack/repos/builtin/packages/votca-csg-tutorials/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class VotcaCsgTutorials(CMakePackage):
+    """Versatile Object-oriented Toolkit for Coarse-graining
+       Applications (VOTCA) is a package intended to reduce the amount of
+       routine work when doing systematic coarse-graining of various
+       systems. The core is written in C++.
+
+       This package contains the VOTCA coarse-graining tutorials.
+    """
+    homepage = "http://www.votca.org"
+    url      = "https://github.com/votca/csg-tutorials/tarball/v1.4"
+    git      = "https://github.com/votca/csg-tutorials.git"
+
+    version('develop', branch='master')
+    version('1.6_rc1', sha256='87c4d945d2bdcb247e985cd407b0767c441f7810f1237ae65a63617f136e2ac9')
+    version('1.5.1',   sha256='e35cea92df0e7d05ca7b449c1b5d84d887a3a23c7796abe3b84e4d6feec7faca')
```

Comment:
Should probably mark this version as `preferred=True` so we don't default to a release candidate.

Reply:
Good point, fixed!